Frameworks/Cedar.xcframework/ios-arm64/Cedar.framework/Headers/CDRSpec.h

```diff
@@ -16,24 +16,30 @@ extern const __nullable CDRSpecBlock PENDING;
 #ifdef __cplusplus
 extern "C" {
 #endif
-void beforeEach(CDRSpecBlock);
+
+    void beforeEach(CDRSpecBlock);
```

Comment:
nit: unexpected spaces at the beginning of the line

Reply:
done